cli/src/commands/squash.rs

```diff
@@ -65,15 +64,20 @@ pub(crate) struct SquashArgs {
     /// description(s) of the source revision(s)
     #[arg(long, short, conflicts_with = "message_paragraphs")]
     use_destination_message: bool,
-    /// Interactively choose which parts to squash
+    /// Interactively choose which parts to squash.
+    /// Implies --keep-empty.
```

Comment:
rewording nit to keep it consistent with other args:

```suggestion
    /// Interactively choose which parts to squash (implies --keep-empty)
```